src/tqec/detectors/operation.py

```diff
@@ -0,0 +1,244 @@
+from typing import Any
+from dataclasses import dataclass
+
+import cirq
+
+STIM_TAG = "STIM_OPERATION"
+
+
+class ShiftCoords(cirq.Operation):
+    def __init__(self, *shift: int) -> None:
```

Comment:
```suggestion
    def __init__(self, *shifts: int) -> None:
```
In the code, `shift` is a `tuple[int, ...]` and so might contain several shifts. It might be easier to understand if this is clear from the variable name.